services/gateway/apis/http-version3/methods/events.js

```diff
@@ -35,8 +35,15 @@ module.exports = {
 		sort: {
 			optional: true,
 			type: 'string',
-			enum: ['timestamp:asc', 'timestamp:desc'],
-			default: 'timestamp:desc',
+			enum: [
+				'height:asc',
+				'height:desc',
+				'index:asc',
+				'index:desc',
```

Comment:
Kindly add these options as `order` param and merge it internally to the `sort` param. `sort` should take precedence over the `order` param. Also, please update the swagger 🙏